cloudformation/ci.template.js

```diff
@@ -70,6 +70,20 @@ AWSTemplateFormatVersion: '2010-09-09',
                 }
               ]
             }
+          },
+          {
+            PolicyName: 'publish-metrics',
+            PolicyDocument: {
+              Statement: [
+                {
+                  Action: ['s3:PutObject'],
+                  Effect: 'Allow',
+                  Resource: ['arn:aws:s3:::mapbox-loading-dock/raw/mobile_staging.codecoverage_v2/*',
+                             'arn:aws:s3:::mapbox-loading-dock/raw/mobile.codecoverage/*',
+                             'arn:aws:s3:::mapbox-loading-dock/raw/mobile_staging.codecoverage/*']
+                }
+              ]
+            }
```

Comment:
did you already run the formation update job for this?

Peng just opened [this](https://github.com/mapbox/mapbox-maps-internal/issues/782) ticket for moving cloud formation to private repo. Maybe we can take this PR as an opportunity to JFDI

Reply:
Yep, update job is run. 

Let's address moving cloud formation later; it'll be a sizable job and affects multiple steps.